src/resultsdisassemblypage.cpp

```diff
@@ -221,11 +221,11 @@ ResultsDisassemblyPage::ResultsDisassemblyPage(CostContextMenu* costContextMenu,
     , ui(std::make_unique<Ui::ResultsDisassemblyPage>())
 #if KFSyntaxHighlighting_FOUND
     , m_repository(std::make_unique<KSyntaxHighlighting::Repository>())
-    , m_disassemblyModel(new DisassemblyModel(m_repository.get(), this))
-    , m_sourceCodeModel(new SourceCodeModel(m_repository.get(), this))
+    , m_disassemblyModel(new DisassemblyModel(m_repository.get(), 8, this))
```

Comment:
here too - please introduce a central `DefaultTabWidth` somewhere and use it everywhere